web_src/js/utils.js

```diff
@@ -142,3 +144,7 @@ export function serializeXml(node) {
 }
 
 export const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+
+export function isWellKnownImageFilename(fn) {
+  return /\.(jpe?g|png|gif|webp|svg)$/i.test(fn);
```

Comment:
```suggestion
  return /\.(jpe?g|png|gif|webp|svg|heic)$/i.test(fn);
```

Somewhat common in Apple ecosystem.